### PR TITLE
added an example of source element using height & width attributes

### LINKED ISSUE #20927: [HTML] Implement width/height attributes on source elements
## Acceptance Criteria

- [x] The listed features are documented sufficiently on MDN
- [x] BCD is updated
- [x] Interactive example and data repos are updated if appropriate
- [x] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [x] Set bugs to `dev-doc-complete`
- [x] Add entry to Firefox release notes if feature is enabled in release
- [x] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Related Gecko bugs

https://bugzilla.mozilla.org/show_bug.cgi?id=1694741

## Other

- [ ] Check content open issues to see if any pertain to the subject matter. If there are any that can be closed because of the work, do so. If there are any that can be fixed relatively quickly because of the knowledge from completing this issue and you have time, feel free to go ahead and fix them.
- [x] Check if [Glossary](https://developer.mozilla.org/en-US/docs/Glossary) updates are required for the feature you're documenting - whether an existing term needs to be updated or a new term should be added.

### PR DESCRIPTION
**Pull request by dletorey**
### Description

Added an example of how the height/width attributes work on the <source> element as a child element of <picture>

### Motivation

Working on [Content Issue 20927](https://github.com/mdn/content/issues/20927)

Also adds extra example of how these attributes are used

### Additional details

- [Bugzilla 1694741](https://bugzilla.mozilla.org/show_bug.cgi?id=1694741)
- [Content Issue #20927](https://github.com/mdn/content/issues/20927)
- [Test example](https://codepen.io/CodeRedDigital/pen/OJEJbaa)

### Related issues and pull requests

- [Fixes #20927](https://github.com/mdn/content/issues/20927)
- [Dependant on Browser Compat Data update #18081](https://github.com/mdn/browser-compat-data/pull/18081)

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/107/index.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ This article provides information about the changes in Firefox 107 that will aff
 
 ### HTML
 
+- The [`height`]() & [`width`]() attributes have been added to {{HTMLElement("source")}} when it is a child of the {{HTMLElement("picture")}}.
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/107/index.md:21]: Should these changes be added here?

File: files/en-us/web/html/element/source/index.md [hunk 2]
```diff
@@ -161,6 +161,22 @@ In this example, two `<source>` elements are included within the {{HTMLElement("
 </picture>
 ```
 
+### Picture with height & width attributes example
+
+In this example, three `<source>` elements are included within the {{HTMLElement("picture")}}, each `<source>` element has it's own {{htmlattrdef("height")}} & {{htmlattrdef("width")}} attributes applied and this help reduce Layout Shift.
```
Comment (bsmth) [inline files/en-us/web/html/element/source/index.md:166]:
```suggestion
In this example, three `<source>` elements with {{htmlattrdef("height")}} and {{htmlattrdef("width")}} attributes are included in a {{HTMLElement("picture")}} element.
A [media query](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) allows the browser to select an image to display with the `height` and `width` attributes based on the [viewport](/en-US/docs/Glossary/Viewport) size.
```

File: files/en-us/web/html/element/source/index.md [hunk 3]
```diff
@@ -161,6 +161,22 @@ In this example, two `<source>` elements are included within the {{HTMLElement("
...
+media="(min-width: 1000px)" width="1000" height="400">
+  <source srcset="mdn-logo-square.png"
+media="(min-width: 800px)" width="800" height="800">
+  <source srcset="mdn-logo-portrait.png"
+media="(min-width: 600px)" width="600" height="800">
+<img src="mdn-logo-fallback.png " alt="" width="500" height="400">
+</picture>
+```
+
 With the `<picture>` element, you must always include an `<img>` with a fallback image, with an `alt` attribute to ensure accessibility (unless the image is an irrelevant background decorative image).
```
Comment (bsmth) [inline files/en-us/web/html/element/source/index.md:180]: I think this one was part of the previous example, not sure if it makes a difference

File: files/en-us/web/html/element/source/index.md [hunk 4]
```diff
@@ -161,6 +161,22 @@ In this example, two `<source>` elements are included within the {{HTMLElement("
 </picture>
 ```
 
+### Picture with height & width attributes example
+
+In this example, three `<source>` elements are included within the {{HTMLElement("picture")}}, each `<source>` element has it's own {{htmlattrdef("height")}} & {{htmlattrdef("width")}} attributes applied and this help reduce Layout Shift.
+
+```html
+<picture>
```
Comment (bsmth) [inline files/en-us/web/html/element/source/index.md:169]: Can you run Prettier over the example?

File: files/en-us/web/html/element/source/index.md (hunk 3 above)
Comment (dletorey) [reply files/en-us/web/html/element/source/index.md:181]: Yes it was part of the original example, but it applies to both which is my justification for moving it below both examples

Comment (dletorey) [reply files/en-us/web/html/element/source/index.md]: I have re-ordered it

File: files/en-us/web/html/element/source/index.md (hunk 4 above)
Comment (dletorey) [reply files/en-us/web/html/element/source/index.md:172]: Done

File: files/en-us/web/html/element/source/index.md [hunk 5]
```diff
@@ -161,6 +161,22 @@ In this example, two `<source>` elements are included within the {{HTMLElement("
...
+  <source srcset="mdn-logo-landscape.png"
+media="(min-width: 1000px)" width="1000" height="400">
+  <source srcset="mdn-logo-square.png"
+media="(min-width: 800px)" width="800" height="800">
+  <source srcset="mdn-logo-portrait.png"
+media="(min-width: 600px)" width="600" height="800">
+<img src="mdn-logo-fallback.png " alt="" width="500" height="400">
```
[the remark this answers is not in the document]
Comment (dletorey) [reply files/en-us/web/html/element/source/index.md]: done

File: files/en-us/mozilla/firefox/releases/107/index.md (hunk 1 above)
Comment (dletorey) [reply files/en-us/mozilla/firefox/releases/107/index.md:21]: Shall remove

File: files/en-us/mozilla/firefox/releases/106/index.md [hunk 6]
```diff
@@ -16,6 +16,9 @@ This article provides information about the changes in Firefox 106 that will aff
 
 ### HTML
 
+- The {{HTMLElement("source")}} element now supports [`height`](/en-US/docs/Web/HTML/Element/source#attr-height) & [`width`](/en-US/docs/Web/HTML/Element/source#attr-width) attributes when it is a child of a {{HTMLElement("picture")}} element.
+  This functionality is an experimental feature behind a [feature flag](/en-US/docs/Mozilla/Firefox/Experimental_features#height_width_attributes_for_source) ({{bug(1694741)}}).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/106/index.md:20]:
```suggestion
  This functionality is an experimental feature enabled using the `dom.picture_source_dimension_attributes.enabled` [preference](/en-US/docs/Mozilla/Firefox/Experimental_features#height_width_attributes_for_source) ({{bug(1694741)}}).
```

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 7]
```diff
@@ -143,6 +143,48 @@ HTML password input elements ([`<input type="password">`](/en-US/docs/Web/HTML/E
   </tbody>
 </table>
 
+### Height & Width attributes for `<source>`
+
+HTML attributes for [height](/en-US/docs/Web/HTML/Element/source#attr-height) & [width](/en-US/docs/Web/HTML/Element/source#attr-width) ({{bug(1694741)}}).
+
+This works when the {{HTMLElement("source")}} when it is a child element of {{HTMLElement("picture")}} and helps eliminate layout shift.
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:150]:
```suggestion
These attributes can be used in {{HTMLElement("source")}} elements when they are children of {{HTMLElement("picture")}} elements and help to eliminate layout shift.
```

**Review by bsmth — APPROVED** [review]
Two open comments to resolve, otherwise looks good, thanks 👍🏻

**Comment by bsmth** [comment]
Thanks @dletorey 🚀

**Pull request merged**